src/app/scripts/cac/control/cac-control-explore.js

```diff
@@ -405,6 +414,21 @@ CAC.Control.Explore = (function (_, $, Geocoder, MapTemplates, HomeTemplates, Ro
         displayPlaces(destinations, $(options.selectors.isochroneSlider).val());
     }
 
+    /**
+     * Filter destinations by category client-side.
+     */
+    function filterPlaces(allPlaces, filter) {
+        if (filter === 'All') {
```

Comment:
When I load the home page and click "Map View", I get this:
![image](https://user-images.githubusercontent.com/6598836/33724651-fde577fc-db3d-11e7-9c90-6b9b811bebb2.png)

Because `filter` is undefined when it gets here. (If I click "All" it then loads the destinations.)

Reply:
This is just on the map view, correct? #933 exists for the map view filter.

Reply:
Yeah, just on the map view.  I was interpreting #933 as just about the interface, but yeah, this can be handled there.